config/samples/perf_v1alpha1_sysbench.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: perf.kubestone.xridge.io/v1alpha1
+kind: Sysbench
+metadata:
+  name: sysbench-sample
+spec:
+  image:
+    name: xridge/sysbench:latest
```

Comment:
would it make sense to use an exact version here? maybe tagged with sysbench version?

Reply:
Done

Reply:
Done

Reply:
I updated the samples to have fixed versions there.